Sources/KituraNet/HTTP/HTTPRequestHandler.swift

```diff
@@ -93,7 +93,7 @@ internal class HTTPRequestHandler: ChannelInboundHandler, RemovableChannelHandle
                     } catch {
```

Comment:
you also need to bump the minimum NIO version in `Package.swift` to `from: "2.12.0"` because you're using new stuff from 2.12